Scripts/script-urlscan-get-http-transactions.yml

```diff
@@ -0,0 +1,52 @@
+commonfields:
+  id: urlscan-get-http-transactions
+  version: -1
+name: urlscan-get-http-transactions
+script: |-
+  var url = args.url;
+  var limit = args.limitl;
+  var defaultWaitTime = 5
+
+  uuid = executeCommand('urlscan-submit-url-command', {'url': url})[0].Contents;
+  uri = executeCommand('urlscan-get-result-page', {'uuid': uuid})[0].Contents;
+
+  var resStatusCode = -1
+  var waitedTime = 0
+  while(resStatusCode != 200 && waitedTime < args.timeout) {
+      var resStatusCode = executeCommand('urlscan-poll-uri', {'uri': uri})[0].Contents;
+
+      if (resStatusCode == 200) {
+          break;
+      }
+      wait(defaultWaitTime);
+      waitedTime = waitedTime + defaultWaitTime;
+  }
+  if(resStatusCode == 200) {
+      return executeCommand('urlscan-get-http-transaction-list', {'uuid': uuid, 'url': url, 'limit': limit});
+  } else {
+      return 'Could not get result from UrlScan, please try again later.'
```

Comment:
print the error, check if there is an error In the request or it is timeout.
if timeout, print a message that asks the user to increase the timeout,